README.rst

```diff
@@ -0,0 +1,104 @@
+===== 
+Frontend Component: Site Header 
+===== 
+A reusable react site header component for edX.org
+
+
+Usage
+--------
+Install the component in your project:
+
+``npm i --save @edx/frontend-component-site-header``
+
+
+In your code:
+
+.. code-block:: javascript
+
+  import { SiteHeader } from "@edx/frontend-component-site-header";
+  
+  // ...code...
+
+  render() {
```

Comment:
Would it be easier to keep this up to date if we put a pointer to the example code itself, instead?

Reply:
Good point

Reply:
I think I'll leave that until the next update, but that seems like the way to go.